confidential-data-hub/kms/Cargo.toml

```diff
@@ -13,6 +13,7 @@ bincode = { workspace = true, optional = true }
 chrono = { workspace = true, optional = true }
 const_format.workspace = true
 crypto = { path = "../../attestation-agent/deps/crypto", optional = true }
+ehsm_client = {git = "https://github.com/intel/ehsm", optional = true }
```

Comment:
It is good to have a `rev` here to fix the version of the upstream version, which is helpful to avoid breaking caused by api changes

Reply:
That's helpful. Thanks!

Reply:
: )  Could you help to squash the commits into one?

Reply:
Sure thing. : )